nss-day-cohort-45/tabloidfullstack-giant-gibbons
# Cjd addpost

Done: User may add a new post

To test:
log in, select "add post" from navbar, enter a post (make sure your image URL is under 255 characters - use tinyurl if necessary)
Select Save and be taken to the list of posts with yours at the top
